tensorflow/lite/delegates/nnapi/nnapi_delegate_kernel.h

```diff
@@ -185,6 +185,9 @@ class NNMemory {
   size_t byte_size_ = 0;
   uint8_t* data_ptr_ = nullptr;
   ANeuralNetworksMemory* nn_memory_handle_ = nullptr;
+#ifndef __ANDROID__
+  std::string shm_region_name;
```

Comment:
hmm, ClangTidy is complaining about the class field variable name, which blocks auto-merger.

Could you change it to "shm_region_name_" ?

Reply:
Thanks, I renamed the variable.